### PR TITLE
About the different types of Status Communities

### PR DESCRIPTION
**Pull request by cheny0**
- First draft Rebase develop
- Ready for review

### CODE REVIEW

**Review by jorge-campo — CHANGES_REQUESTED** [review]
Hey @cheny0 – Thanks for your article!

I have some comments that can improve the table. Let me know if they don't make sense.

- 'In Status, there are different types of communities, and your ability to access them varies. Your access can be affected by three factors listed in the table:'. In addition to the table changes I propose below, I would say, 'In Status, there are different types of communities, and your ability to access them depends on three factors: token requirements, join requirements and discoverability.

I propose these changes to your table:

| Community | Access requirement | Managed by | Note |
|:---|:---|:---|:---|
| Token-gated | [Token requirements][understand-token-requirements-in-communities] | Community owner and admins | You need to hold the required tokens to join. |
| Open | None | N/A | You are not required to hold any specific tokens to join.|
| Join approval required | [Join approval][about-community-request-approvals] | Community owner and admins | You need to wait for the community owner or admin to approve your join request. |
| Join approval not required | None | N/A | You can join the communities directly if you meet the token requirements or there's no token requirement. |
| Private | You need to know the community’s public key to discover the community. | SNT token holders | The community doesn’t appear on the Discover page in Communities.</br> SNT holders can vote to [change the community’s visibility][about-voting-to-change-community-visibility]. |
| Public | None | N/A | The community appears on the Discover page in Communities</br>. SNT holders can vote to [change the community’s visibility][about-voting-to-change-community-visibility]. |

Notes:

 - I think you've mixed the Private and Public communities' notes in your table (?).
 - In the Private/Public, I'm changing things a bit so the requirement or factor is now knowing the public's community key. And in the 'Note' column, I added information about the voting system.
 - Transform your Note-style admonition into a standard paragraph and add a note-style admonition with your sentence, 'When a new community is created, it's private by default.'

Other comments:

- You don't need the first H2. Your first paragraph and the table are part of your introduction; we don't use headers for introductions. See the structure of topics in our writing style guide for more details. (https://write.status.im/structuring-the-content/#concept-structure)
- Note admonition – I'd start by saying, '_These factors work independently_ and your access to a community depends...'

Examples of different communities:

- First example: 
    - 'In this case, the artist's community could be closed and public.' -A 'In this case, the artist's community could be _token-gated_ and public'.

- Second example:
    - Instead of 'employer' or 'company', use 'organization' to broaden the example's scope.
    - 'Your company could choose to create a community that is open and private, and it requires approval for join requests. In this way, your employer could verify users' identities before they join and would not receive irrelevant join requests. – I think this paragraph needs an update.
      - 'Your company could choose to create a community that is open and private, and it requires approval for join requests' -> What do you think of 'Your organization could choose to create an open and private community, and require approval for join requests' ?
      - If the community requires approval for a join request, the community owner _may_ still receives irrelevant join requests if someone knows the community's public key. I think a better option for any organization is to keep the community without join requests but token-gate the community. The fact that it's a public or private one is secondary.

**Comment by cheny0** [comment]
Hi @jorge-campo, I have implemented your comments. Thank you!

**Pull request closed without merging**
